.storybook/index.js

```diff
@@ -44,17 +44,23 @@ const getComponentsCategory = bindCategory(COMPONENTS);
  */
 const getPatternsCategory = bindCategory(PATTERNS);
 
+/**
+ * Configuration for disabling the centering addon.
+ * @returns {Object<string, Object>} The disabled configuration.
+ */
+const disableCentered = () => ({
+  centered: {
+    disable: true,
+  },
+});
+
 /**
  * Configuration for disabling the centering addon in individual stories.
  * @param {Object<string, Object>} stories The collection of stories to disable the addon for.
  * @returns {Object<string, Object>} The collection of stories.
  */
-const disableCentered = stories =>
-  stories.addParameters({
-    centered: {
-      disable: true,
-    },
-  });
+const disableCenteredStories = stories =>
+  stories.addParameters(disableCentered());
```

Comment:
Is there any way to leave out this change to reduce the subset of changes + files edited in this PR? 😅 
It is producing quite a bit of noise so I was wondering if it is necessary here?

Reply:
I could add it in a separate PR if you'd prefer, but at the time it was required for this work to disable `@storybook/addon-centered` for both the old story format and CSF